charts/redpanda/README.md

```diff
@@ -34,6 +34,37 @@ To learn how to override the default values in the `values.yaml` file, see the [
 
 All configuration options for the Redpanda Helm chart are documented in the [`values.yaml`][values] file.
 
+## Upgrading Chart
+
+```bash
+helm upgrade [RELEASE_NAME] redpanda/redpanda
+```
+
+### From 2.6.x onwards
+
+In order to enable dedicated persistent volume for tiered storage cache, the `storage.tieredStoragePersistentVolume.enabled` need to be set to `true`.
+The `helm upgrade` will fail with the following error.
+```bash
+helm upgrade --namespace redpanda redpanda/redpanda \
+  --set storage.tieredStoragePersistentVolume.enabled=true \
+  --set storage.tieredConfig.cloud_storage_enabled=true \
+  --set storage.tieredConfig.cloud_storage_cache_directory=/some/path/for-tiered-storage
+Error: UPGRADE FAILED: cannot patch "redpanda" with kind StatefulSet: StatefulSet.apps "redpanda" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden
+```
+
+To be able to add PersistentVolume for tiered storage cache please delete StatefulSet with cascade orphan option to leave Pods behind.
+```bash
+kubectl delete sts redpanda --cascade=orphan
+```
+
+The `helm upgrade` should be able to succeed, but you need to manually do rolling update starting from ordinal 0 to ordinal N.
+
+```bash
+kubectl delete pod redpanda-0
+```
+
+Please wait for deleted Pod to be restarted and become ready in order to move to next Pod.
```

Comment:
Thanks for working out a way to perform these types of updates for a helm deployment. But how will recommending user delete the statefulset change their ongoing interactions with the cluster? I assume that from now on they will need to manually perform rolling updates from this point forward?

Reply:
Not at all. If user want to have tiered storage cache backed by persistent volume, then user would be required to perform manual deletion of each pod to roll out new pod definition. After that all automation build in statefulset and our post install helm hook jobs will run as always. Nothing else will be manual.